sdk/tables/azure-data-tables/azure/data/tables/_serialize.py

```diff
@@ -152,21 +152,13 @@ def _to_entity_none(value):  # pylint: disable=unused-argument
     UUID: _to_entity_guid,
     Enum: _to_entity_str,
 }
-try:
-    _PYTHON_TO_ENTITY_CONVERSIONS.update(
-        {
-            unicode: _to_entity_str,  # type: ignore
-            str: _to_entity_binary,
-            long: _to_entity_int32,  # type: ignore
-        }
-    )
-except NameError:
-    _PYTHON_TO_ENTITY_CONVERSIONS.update(
-        {
-            str: _to_entity_str,
-            bytes: _to_entity_binary,
-        }
-    )
+
+_PYTHON_TO_ENTITY_CONVERSIONS.update(
```

Comment:
We no longer need to call `upate` here, just move these two lines (158-159) up into the dictionary after line 153.

Reply:
Updated